lib/Subscribers/ImportExport/PersonalDataExporters/NewsletterClicksExporter.php

```diff
@@ -49,6 +49,18 @@ private function exportNewsletter($row) {
       'name' => WPFunctions::get()->__('URL', 'mailpoet'),
       'value' => $row['url'],
     ];
+
+    if (!is_null($row['user_agent'])) {
+      $userAgent = $row['user_agent'];
+    } else {
+      $userAgent = WPFunctions::get()->__('Unknown', 'mailpoet');
```

Comment:
You don't have to call `WPFunctions::get()->__` it was done automatically, but we normally use just `__`

Reply:
I will try to remember this. I think in this case I simply copied what was already being used in this file. Why we use `WPFunctions::get()` for all WP functions except `__()`?